app/src/main/java/com/xa/xpensauditor/SetLimitActivity.java

```diff
@@ -148,14 +148,14 @@ public void onClick(View view) {
             public void onDataChange(@NonNull DataSnapshot snapshot) {
                 for (DataSnapshot snapshot3 : snapshot.getChildren()) {
                     if (Integer.parseInt(snapshot3.child("Day").getValue().toString()) == (thisDay)) {
-                        String amm = (snapshot3.child("Amount").getValue()).toString();
+                        String amm = (snapshot3.child("Amount").getValue().toString());
                         int am1 = Integer.parseInt(amm);
                         day_tol = am1 + day_tol;
                     }
                 }
 
                 System.out.println("Day-total " + day_tol);
-                if (dt > Integer.parseInt(sharedPreferences.getString(DLIMIT,"0"))){
+                if (day_tol > Integer.parseInt(sharedPreferences.getString(DLIMIT,"0"))){
```

Comment:
⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.WhitespaceAroundCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>WhitespaceAround: '{' is not preceded with whitespace.